pandas/tests/io/test_sql.py

```diff
@@ -855,6 +855,12 @@ def test_get_schema(self):
         create_sql = sql.get_schema(self.test_frame1, "test", con=self.conn)
         assert "CREATE" in create_sql
 
+    def test_get_schema_with_schema(self):
+        create_sql = sql.get_schema(
+            self.test_frame1, "test", con=self.conn, schema="pypi"
+        )
+        assert "pypi" in create_sql
```

Comment:
Could we make the test a little more robust here. ie. check just the table name is prefixed with the schema

Reply:
@alimcmaster1 
Thanks so much for reviewing my pr, I have changed the test case and rebased from master.